pandas/core/frame.py

```diff
@@ -1815,13 +1815,17 @@ def info(self, verbose=None, buf=None, max_cols=None, memory_usage=None,
         """
         Concise summary of a DataFrame.
```

Comment:
Can you please elaborate more this comment or provide what you think fits the best? Original docstring remains unchanged here.

Reply:
It is just the recommendation from the pandas doscring guide to short summaries:
"For functions and methods, the short summary must start with an infinitive verb."
https://python-sprints.github.io/pandas/guide/pandas_docstring.html

Maybe it would go better with "Display a summary", "Present", "Print" or something like that.